src/sentry/api/endpoints/organization_searches.py

```diff
@@ -13,7 +17,14 @@
 from sentry.models.search_common import SearchType
 
 
+class OrganizationSearchSerializer(serializers.Serializer):
+    type = serializers.IntegerField(required=True)
+    name = serializers.CharField(required=True)
+    query = serializers.CharField(required=True)
```

Comment:
Do we want to ensure that the query has some bytes? Is it useful to save empty queries?

Reply:
Yeah, probably worthwhile